src/type_translator.ts

```diff
@@ -9,6 +9,25 @@
 import * as path from 'path';
 import * as ts from './typescript';
 
+/**
+ * TypeScript allows you to write identifiers quoted, like:
+ *   interface Foo {
+ *     'bar': string;
+ *     'complex name': string;
+ *   }
+ *   Foo.bar;  // ok
+ *   Foo['bar']  // ok
+ *   Foo['complex name']  // ok
+ *
+ * In Closure-land, we want identify that the legal name 'bar' can become an
+ * ordinary field, but we need to skip strings like 'complex name'.
+ */
+export function isValidClosurePropertyName(name: string): boolean {
```

Comment:
This is a weird place for this function.  But its previous home was in a jsdoc file which is also a weird location.  I don't feel great about it.